examples/scenarioSmallBodyLandmarks.py

```diff
@@ -0,0 +1,571 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2016, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
```

Comment:
update 2016 to be the current year. :-)

Reply:
:) who's gonna trust people doing space software that does not know what year it is